### PR TITLE
Settings layout changes improvements

### PR DESCRIPTION
**Pull request by jjd-uk**
This is a refactor of the layout of Settings inspired by the discussions on http://forum.kodi.tv/showthread.php?tid=189031

For Team members I've open a thread in our forums so discussion of the details can be made there to avoid people getting bombarded with too much github spam, as I imagine this will generate a lot of discussion.

To make sure I've not messed up the platform specific xmls (as this has only been tested on Windows) and any other comments @Memphiz @popcornmix @koying @Paxxi @FernetMenta 

Also for UI apsects @da-anda @ronie @phil65 @HitcherUK

### CODE REVIEW

File: addons/skin.confluence/720p/Settings.xml [hunk 1]
```diff
@@ -116,59 +116,47 @@
 				</focusedlayout>
 				<content>
 					<item id="1">
-						<label>480</label>
-						<label2>31400</label2>
-						<onclick>ActivateWindow(AppearanceSettings)</onclick>
+						<label>14200</label>
+						<label2>31430</label2>
+                        <onclick>ActivateWindow(PlayerSettings)</onclick>
```
Comment (phil65) [inline addons/skin.confluence/720p/Settings.xml]: tabs vs. spaces.

**Comment by phil65** [comment]
Lot of small formatting issues in those XMLs. I would suggest to use tools to auto-format and verify XMLs after bikeshedding has ended. ;)

**Comment by Tolriq** [comment]
If there's a rework can I ask something about the Settings level access ?

I find not very logical the navigation to it right now, you can't press up from 1st line to access it, you need to get to bottom then down, then if you press down again you return to last item and not first item. And if pressing down again you return to settings level.

While when pressing up you can do a loop in all categories but the settings level. This breaks a little the navigation pattern.

**Comment by phil65** [comment]
@Tolriq : that's unrelated to this PR. To get that done properly,  someone would have to implement SetFocus(id,pos) for grouplists.

**Comment by bossanova808** [comment]
If you're going to do all this re-arranging, surely a good time to move the crazily undiscoverable

Skin
- Settings

to somewhere more obvious/sensible.  I've seen people using Kodi for years miss that one...

**Comment by Razzeee** [comment]
Whats the status here?

**Comment by jjd-uk** [comment]
I'm at a stage now where I need feedback from the Team, also I've not yet tested on any platform other than Windows so help with that is required as I've altered the platform specific xml's blindly (android.xml, darwin.xml etc).

I've rebased on master as it was yesterday and just waiting for jenkins (http://jenkins.kodi.tv/view/Helpers/job/BuildMulti-PR-Manually/213/) to complete so I can post some up to date test builds to the forum.

**Comment by DaveTBlake** [comment]
Plenty of +1 on the forum for this change. Can we get it rebased and merged?

**Comment by jjd-uk** [comment]
jenkins build this please

**Comment by jjd-uk** [comment]
@phil65 Martijn was keen to get this in so it's been merged with some Estuary changes I did to allow testing, so you may want to review them. 

- For the icons I just reused what you already had for the skin, so you may wish to change these.

- For the ordering I put what I felt were like items together, so top row is the media related items, then second row I wanted to keep Interface (as this is all things related to the GUI) and Skin settings together.

- For the strings I've added two versions of the strings for each settings category e.g. **"Library"** and **"Library settings"** so skins can choose whatever they want. I partly did this since in Estuary we have non-settings stuff like **"System Info"**, so I wanted to clearly distinguish between that and **"System"** settings, plus since you moved **"Skin settings"** to be in the main category list I thought it added consistency.

You however may disagree so do whatever you feel is appropriate as I'm no skinner.

**Comment by FernetMenta** [comment]
well done!

SETTING_VIDEOPLAYER_PAUSEAFTERREFRESHCHANGE is obsolete

**Comment by jjd-uk** [comment]
Thanks FernetMenta. 

I did try and follow everything that was getting changed in Master each time I was rebasing. Let me know if there's anything else that needs to be addressed.

**Comment by phil65** [comment]
I will take a look at skin-related stuff, thx for info.

**Comment by AlwinEsch** [comment]
Hi @MartijnKaijser was a bit hard to find :) but why you have changed DSP enable/disable to level 3
has a bit confused me ;)
```xml
        <setting id="audiooutput.dspaddonsenabled" type="boolean" label="36441" help="36438">
          <level>2</level>
          <default>false</default>
          <dependencies>
            <dependency type="visible">
              <or>
                <condition on="property" name="aesettingvisible" setting="audiooutput.passthrough">audiooutput.dspaddonsenabled</condition>
                <condition on="property" name="aesettingvisible" setting="audiooutput.ac3passthrough">audiooutput.dspaddonsenabled</condition>
              </or>
            </dependency>
          </dependencies>
```

**Pull request merged**